evap/staff/templates/staff_course_comments_quick.html

```diff
@@ -224,14 +224,29 @@ <h5 class="modal-title" id="hotkeys-modal-title">Hotkeys</h5>
             updateState();
         }
 
-        function slide(direction) {
-            if(direction === "left" && index > 0) {
+        function slide(to) {
+            let direction = to;
+            if(to === "left" && index > 0) {
                 index--;
-            } else if(direction === "right" && index < items.length) {
+            } else if(to === "right" && index < items.length) {
                 index++;
+            } else if(Number.isInteger(to)) {
+                direction = to >= index? "right" : "left";
+                index = to;
```

Comment:
cool, you found a more general case of the slide method, which is to give it an arbitrary index instead of "left" or "right". this makes the "left" and "right" parameters look a bit like unnecessary convenience, how about we use the general case everywhere? so, instead of calling `slide("right")`, one would call `slide(index+1)`?

Reply:
Okay, if they will underflow `index < 0` or overflow `index > items.length` then I can just reset them to bounds. Nice thing which I didn't had planned, but if you prefer an index driven interface rather than a string interface, it's fine on my side